libpod/container_config.go

```diff
@@ -364,4 +364,6 @@ type ContainerMiscConfig struct {
 	Timezone string `json:"timezone,omitempty"`
 	// Umask is the umask inside the container.
 	Umask string `json:"umask,omitempty"`
+	// PidFile is the file that saves the pid of the container process
+	PidFile string
```

Comment:
Needs a JSON tag with `omitempty`

Reply:
Have added json tag. I am a beginner developer for Podman, sorry for these rookie mistakes

Reply:
No problem, thanks for contributing!